polynote-frontend/polynote/ui/component/notebook/kernel.ts

```diff
@@ -359,6 +360,14 @@ class KernelTasksEl extends Disposable {
             }
         }
     }
+
+    private jumpToCell(id: string) {
+        const nbInfo = ServerStateHandler.getOrCreateNotebook(this.notebookPathHandler.state);
+        const idNum = id.split(" ").pop(); // extract the actual id number
+        if (!idNum) return; // pop can return undefined - if it does, do nothing
```

Comment:
Could we instead do `if(idNum) <work>` - writing for the happy path 😄

Reply:
Also, just to be safe we should check for undefined explicitly rather than just falsiness